dataactvalidator/sqs_work_dispatcher.py

```diff
@@ -187,7 +187,7 @@ def signal_reset_wrapper(func, args, kwargs):
             name=self.worker_process_name,
             target=signal_reset_wrapper,
             args=(job, job_args, job_kwargs),
-            daemon=True  # daemon=True ensures that if the parent dispatcher dies, it will kill this child worker
+            daemon=False  # daemon=True ensures that if the parent dispatcher dies, it will kill this child worker
```

Comment:
One risk with this is that if the parent process does go down unexpectedly, child worker processes could keep on running. And considering the work they do (like validation), it could be resource-intense. 
On the USAS bulk downloader side, we had to set this to false too (since workers spawn psql and zip subprocesses) ... but to compensate, we put in a bunch of extra cleanup code in with `try-finally` blocks around areas that could potentially error-out, which attempted to find and destroy the child worker processes upon exit of the parent. 

Please look to add some similar cleanup to any try-finally wrappers in this code (and add the try-finally wrappers if missing).

Example cleanup functions and usages here: https://github.com/fedspendingtransparency/usaspending-api/commit/bc70a443f3eb0afa0b53b36ae5cd8b2de6f55544

Reply:
Updated SQSDispatcher and updated ValidationManager to close all child processes upon completion.